src/main/java/org/folio/oaipmh/client/SourceStorageSourceRecordsClient.java

```diff
@@ -69,8 +67,14 @@ public void postSourceStorageSourceRecords(String idType, Boolean deleted, List
       buffer.appendString(ClientHelpers.pojo2json(List));
     }
 
+
     HttpClientRequest request = this.httpClient.postAbs(this.okapiUrl + "/source-storage/source-records" + queryParams.toString());
     request.handler(responseHandler);
+    request.exceptionHandler(e-> {
+      logger.error("SRS response error 1234: " + e.getMessage(), e);
```

Comment:
what is the significance of error 1234?

Reply:
```suggestion
      logger.error("Exception making SRS request: " + e.getMessage(), e);
```

Reply:
Thanks for the note, initially this log was added just for testing of implemented logic work. I deleted it since we have loggin in the "srs retry method".